src/v/redpanda/admin/api-doc/security.def.json

```diff
@@ -21,4 +21,101 @@
             "type": "string"
         }
     }
+},
+"role_definition": {
+    "type": "object",
+    "properties": {
+        "role": {
+            "type": "string"
+        }
+    }
+},
+"role_description": {
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        }
+    }
+},
+"role_member": {
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "principal_type": {
+            "type": "string"
+        }
+    }
+},
+"role_members_list": {
+    "type": "object",
+    "properties": {
+        "members": {
+            "type": "array",
+            "items": {
+                "type": "role_member"
```

Comment:
```suggestion
                "$ref": "role_member"
```

Reply:
man, I would have sworn that using "$ref" like this was blowing up the code gen script! seems to work now. was months ago that I was messing with this...

Reply:
done